jib-gradle-plugin/src/integration-test/resources/gradle/projects/empty/build.gradle

```diff
@@ -26,5 +26,5 @@ jib {
 
 def additionalTag = System.getProperty("_ADDITIONAL_TAG")
 if (additionalTag != null && !additionalTag.isEmpty()) {
-  jib.to.tags = [System.getProperty("_ADDITIONAL_TAG")]
+  jib.to.tags = [System.getProperty("_ADDITIONAL_TAG")] as Set
```

Comment:
Hmm... I think we should keep it backward-compatible. It will be common to use a list. I suspect that adding another `setTags()` stops Gradle from auto-casting in to a `Set`. @loosebazooka maybe we need to add a `setTags(List)` too?

Reply:
You're absolutely right. I just realized that this change doesn't really solve the backward-compatibility issue.